src/Coypu.AcceptanceTests/ApiExamples.cs

```diff
@@ -646,6 +646,7 @@ private static RemoteWebDriver CustomProfile()
 
         [TestCase("Windows 7", "firefox", "25")]
         [TestCase("Windows XP", "internet explorer", "6")]
+        [TestCase("Windows 10", "MicrosoftEdge", "14")]
         public void CustomBrowserSession(string platform, string browserName, string version)

```

Comment:
Hi, Yes I did. The way the test is implemented it doesn't require changes to the DriverFactory.cs as in this case it is passing in the RemoteWebDriver in the constructor so it is not making use of the DriverFactory.cs, which is the same case that I have in my codebase. There is no harm in adding it though. I can submit another pull request with the changes and remove the commits of the bump of the version and the history.md changes if you prefer.